lib/config.js

```diff
@@ -148,9 +148,11 @@ function readConfigFromFile(filePath) {
             filePath = "eslint-config-" + filePath;
         }
 
-        // TODO: support other configs
-        // config = util.mergeConfigs(config, require(filePath));
+        if (filePath.match(/^eslint-config-airbnb.*/)) {
```

Comment:
I think we'd have to do that recursively and check the actual config it's generating if we wanted to do it here. The only plugin they currently require is `react`.

If we did want to check it still, how would we want to go about doing that?